packages/rum-core/src/domain/trackSleep.ts

```diff
@@ -0,0 +1,39 @@
+import { monitor, ONE_MINUTE, ONE_SECOND, TimeStamp, timeStampNow } from '@datadog/browser-core'
+
+export const SLEEP_CHECK_DELAY = ONE_SECOND
+export const SLEEP_THRESHOLD = ONE_MINUTE
+
+let sleepPeriods: Array<{ start: TimeStamp; end: TimeStamp }> | undefined
+let lastWokeDate: TimeStamp | undefined
+
+export function trackSleep() {
+  lastWokeDate = timeStampNow()
```

Comment:
nit: 
```suggestion
  lastWoke = timeStampNow()
```

to be consistent with other naming (now, start, end)